CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_netsuite_source v0.10.0
+[PR #47](https://github.com/fivetran/dbt_netsuite_source/pull/47) includes the following update:
+## 🚨 Breaking Changes 🚨
+- Added column `_fivetran_synced_date` to model `stg_netsuite2__transactions` for use in downstream models.
+- To reduce storage, updated default materialization of staging models to views. 
+>  ⚠️ Running a `--full-refresh` will be required if you have previously run these staging models as tables and get the following error: 
+> ```
+> Trying to create view <model path> but it currently exists as a table. Either drop <model path> manually, or run dbt with `--full-refresh` and dbt will drop it for you.
+> ```
+
+## Under the Hood:
+- Added integration testing pipeline for Databricks SQL Warehouse.
+- Included auto-releaser GitHub Actions workflow to automate future releases.
```

Comment:
i don't actually see the auto-releaser workflow files in this PR/[branch](https://github.com/fivetran/dbt_netsuite_source/tree/release/v0.10.0-apr-2024/.github)

Reply:
🤦‍♀️ Thank you. Added!